compiler/bindings/python/test/ir/tuner_test.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2025 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from iree.compiler import ir
+from iree.compiler.dialects import iree_codegen
+
+
+def run(fn):
+    with ir.Context(), ir.Location.unknown():
+        module = ir.Module.create()
+        with ir.InsertionPoint(module.body):
+            print("\nTEST:", fn.__name__)
+            fn()
+    return fn
+
+
+@run
+def root_op():
```

Comment:
I think this code should be in a directory outside of `ir`, like `compiler/bindings/python/test/api/tuner_api_test.py`